src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/healthcert/PersonName.kt

```diff
@@ -20,4 +20,14 @@ data class PersonName(
 	@Json(name = "fnt") val standardizedFamilyName: String,
 	@Json(name = "gn") val givenName: String?,
 	@Json(name = "gnt") val standardizedGivenName: String?,
-) : Serializable
\ No newline at end of file
+) : Serializable {
+
+	fun prettyFamilyName(): String = familyName ?: standardizedFamilyName
+
+	fun prettyGivenName(): String = givenName ?: (standardizedGivenName ?: "")
+
+	fun prettyName(): String = "${prettyFamilyName()} ${prettyGivenName()}"
```

Comment:
This and `prettyStandardizedName` could be written with the `joinToString("separator")` list function to deal with the nullability-issue. Not sure if `Mueller<<` should be a correct standardized name if the first name is not present.